src/core/components/wrapper/BaseAnimationWrapper.tsx

```diff
@@ -31,13 +31,13 @@ export abstract class BaseAnimationWrapper<P extends AnimationWrapperProps> exte
         const content = this.props.children;
         if (this.props.animationConfig?.triggerType === AnimationTriggerType.ON_CLICK) {
             return (
-                <TouchableWithoutFeedback onPress={this._onPress}>
+                <TouchableWithoutFeedback onPress={this._onPress} style={this.props?.animationWrapperStyles}>
```

Comment:
Please add a description of this prop, and what do you intend to do with this?

Also, you'll need to update the documentation if you're adding new props to the `BaseAnimationWrapper`